docs/source/deploying-best-practices.rst

```diff
@@ -0,0 +1,70 @@
+Deploying Best Practices
```

Comment:
This doesn't seem as much like a normal best practices doc as it does a general blog post.  It's more of a narrative style.  

I think it makes sense as a blog post and could be good, but I don't think it makes sense here as it is.

Reply:
Also, not that it's super relevant here, but the actual talk was released a couple of weeks ago (albeit with a slight visual lag)

https://www.youtube.com/watch?v=5hUkUj1VYW4

Reply:
I really like the content in this talk. I think this would be valuable to have on a page in the docs (in a less blog-post style)—basically "all the other stuff you have to deal with when deploying dask that you probably haven't thought of yet". Giving people a warning of all the problems they're likely to run into, instead of having to learn the hard way, is very valuable.

I don't think I'd title it "Deploying Best Practices" though. Maybe "Considerations when deploying Dask"?

Reply:
This page has been moved to https://github.com/dask/dask/pull/9933